src/main/java/org/jabref/logic/exporter/ExportFormats.java

```diff
@@ -108,15 +108,19 @@ public static String getConsoleExportList(int maxLineLength, int firstLineSubtr,
     /**
      * Look up the named export format.
      *
-     * @param consoleName
-     *            The export name given in the JabRef console help information.
+     * @param consoleName The export name given in the JabRef console help information.
      * @return The ExportFormat, or null if no exportformat with that name is
-     *         registered.
+     * registered.
      */
     public static IExportFormat getExportFormat(String consoleName) {
         return ExportFormats.EXPORT_FORMATS.get(consoleName);
     }
 
+    public static FileExtensions getFileExtension(String consoleName) {
+        ExportFormat exportFormat = (ExportFormat) EXPORT_FORMATS.get(consoleName);
```

Comment:
What happens if no ExportFormat is found for that console Name?
Then the exportFormat.getExtension will throw an NPE?